packages/client/tests/functional/raw-queries/typed-results/tests.ts

```diff
@@ -6,46 +7,50 @@ declare let prisma: PrismaClient
 declare let Prisma: typeof PrismaNamespace
 
 testMatrix.setupTestSuite(
-  () => {
+  ({ providerFlavor }) => {
     test('simple expression', async () => {
       const result = (await prisma.$queryRaw`SELECT 1 + 1`) as Array<Record<string, unknown>>
       expect(Number(Object.values(result[0])[0])).toEqual(2)
     })
 
-    test('query model with multiple types', async () => {
-      await prisma.testModel.create({
-        data: {
-          id: 1,
-          string: 'str',
-          int: 42,
-          bInt: BigInt('12345'),
-          float: 0.125,
-          bytes: Buffer.from([1, 2, 3]),
-          bool: true,
-          dt: new Date('1900-10-10T01:10:10.001Z'),
-          dec: new Prisma.Decimal('0.0625'),
-        },
-      })
+    // TODO Failing because of "Unsupported type column: 17" error
```

Comment:
```suggestion
    // TODO Failing because of "Unsupported type column: 17" error
    // tracked in https://github.com/prisma/team-orm/issues/374
```